pkg/controller/apikey/create.go

```diff
@@ -66,7 +66,7 @@ func (c *Controller) HandleCreate() http.Handler {
 		apiKey, err := currentRealm.CreateAuthorizedApp(c.db, &authApp, currentUser)
 		if err != nil {
 			if database.IsValidationError(err) {
-				w.WriteHeader(http.StatusBadRequest)
+				w.WriteHeader(http.StatusUnprocessableEntity)
```

Comment:
Just wondering - when should we be using 422 over 400? We have several places that validate input and mostly go with the 400

Reply:
422 generally means "you gave us data, we got the data, we parsed the data, but the data was invalid for some reason (usually validation)"

400 generall means "we didn't understand the request"

https://www.bennadel.com/blog/2434-http-status-codes-for-invalid-data-400-vs-422.htm

Reply:
That being said, we have to keep any _APIs_ at 400 because that's what they're documented as.